api/all.yaml

```diff
@@ -1414,6 +1702,60 @@ components:
           description: |
             Opaque token to retrieve the next page of results. Absent if there are no more pages.
             __page_token__ should be set to this value for the next request (for the next page of results).
+    GenerateTemporaryModelVersionCredential:
+      example:
+        catalog_name: catalog_name
+        schema_name: schema_name
+        model_name: model_name
+        version: version
+        operation: null
+      properties:
+        catalog_name:
+          description: |
+            Catalog name for which temporary credentials are generated. 
+            Can be obtained from models/{full_name} (get model info) API.
+          type: string
+        schema_name:
+          description: |
+            Schema name for which temporary credentials are generated. 
+            Can be obtained from models/{full_name} (get model info) API.
+          type: string
+        model_name:
+          description: |
+            Model name for which temporary credentials are generated. 
+            Can be obtained from models/{full_name} (get model info) API.
+          type: string
+        version:
+          description: |
+            Model version for which temporary credentials are generated.
+          type: integer
+          format: int64
+        operation:
+          $ref: '#/components/schemas/ModelOperation'
+      type: object
+      required:
+        - catalog_name
+        - schema_name
+        - model_name
+        - version
+        - operation
+    GenerateTemporaryModelVersionCredentialResponse:
+      properties:
+        aws_temp_credentials:
+          $ref: '#/components/schemas/AwsCredentials'
+        expiration_time:
+          description: |
+            Server time when the credential will expire, in epoch milliseconds.
+            The API client is advised to cache the credential given this expiration time.
+          type: integer
+          format: int64
+      type: object
+    ModelOperation:
+      enum:
+        - UNKNOWN_MODEL_OPERATION
+        - READ
```

Comment:
Pretty sure these have different names/would collide with tables with the current name

Reply:
Yep....though...do we store the string value of these enums or the integer value?  regardless....fixing.